packages/relayer/indexer/subscribe.go

```diff
@@ -30,23 +30,19 @@ func (svc *Service) subscribe(ctx context.Context, chainID *big.Int) error {
 
 	defer sub.Unsubscribe()
 
-	group, ctx := errgroup.WithContext(ctx)
```

Comment:
we dont need errgroup here as we are not batching, but handling the events as they come in. A block can not support more transactions than our relayer is capable of handling per block, and we were already not calling errgroup.Wait(), so it is fine to remove I thin, and make it cleaner.